docs/events/glides/com.mbta.ctd.glides.editors_changed.v1.mdx

```diff
@@ -1,35 +1,65 @@
 ---
+sidebar_label: glides.editors_changed
 ---
 
-import GlidesExamples from "@site/examples/glides-events-schema/glides-examples.json";
 import EventSchemaPath from "@site/src/components/EventSchemaPath.tsx";
 import Examples from "@site/src/components/Examples.tsx";
+import example from "@site/examples/glides-events/editors_changed.v1.take_over.json";
 
-Operator sign in and out of editing specific trainsheets. This event publishes
-changes to who is editing. Since one action by an inspector might result in
-multiple changes, for example if they sign out of one location and in to
-another, the changes are published in a list.
+# editors_changed
 
-In practice, we limit each location to have only one editor, and limit each
-editor to only one location, but that rule may or may not change and this data
-in this event doesn't guarantee that.
+event type: `com.mbta.ctd.glides.editors_changed.v1`
 
-Note that the author may or may not match the editors in changes, since
-inspectors can change whether other people are editing. For example, when an
-inspector takes over editing at a location, they may sign out the previous
-editor. Then they would issue a stop change for the previous editor and a start
-change for themself.
+Inspectors sign in and out of editing specific trainsheets. This event publishes changes to who is editing.
 
-Fields in the event:
+Since one action by an inspector might result in
+multiple changes, for example if they sign out of one location and in to another, the changes are published in a list.
 
-- `metadata` (Metadata): how the changes were entered in Glides, including the author.
-- `changes` (array of EditorChange): a list of start and stop editing events
+In practice, we limit each location to have only one editor, and limit each editor to only one location, but that rule may or may not change and this event schema doesn't guarantee that.
+
+Note that the author may or may not match the editors in changes, since inspectors can change whether other people are editing.
+For example, when an inspector takes over editing at a location, they may sign out the previous editor.
+Then they would issue a stop change for the previous editor and a start change for themself.
+
+## Data
+
+Fields in the event's `data`:
+
+- `metadata` ([Metadata](.#metadata)): how the changes were entered in Glides, including the author.
+- `changes` (array of [EditorChange](#editor-change)): a list of start and stop editing events.
+
+### EditorChange
+
+Fields:
+
+- `type` (string `"start"`|`"stop"`): Whether the editor started or stopped editing.
```

Comment:
I was wondering about that.

Reasons to put all the documenation in the schema:
- It can all be machine-readable and verified. As is this could have unnoticed errors. Later updates to the schema could especially cause errors.
- Stuff is all in one place. You don't have to jump between prose and schema to correlate the two.
- If someone fetches a schema file, they'd get the documentation too, instead of the documentation being only available on the docs website.

Reasons this is better here:
- Easier to navigate this organized document than clicking through the interactive schema viewer.
- Easier to read paragraphs than string fields.
- Easier to read human-readable descriptions than machine-readable. e.g. `"start"|"stop"` vs `"anyOf": [{"const": "start"}, {"const": "stop"}]`
- Can have richer text. e.g. links to other documentation.
- Can describe things in prose rather than associating each description with a specific field or object in the schema.
- Handling and maintaining docs is easier than handling and maintaining schemas. E.g. there are errors in the dereferencing code that means some documentation would have been clobbered if it were in the schema. Writing text is simpler and just works.

I ended up deciding on keeping the schema focused on validation, and putting the meaning and documentation here where it can be freely organized and formatted and read. But I could be convinced otherwise.